pkg/init/init

```diff
@@ -2,6 +2,14 @@
 
 /bin/mount -t tmpfs tmpfs /mnt
 
+# just copying everything uses a lot of space, so use mv which frees it up
+# copy directory structure under /containers
+find /containers -type d | cpio -p /mnt 2> /dev/null
+# move files under /containers
+find /containers -not -type d -exec mv {} /mnt/{} \;
```

Comment:
I can't detect any slowdown in running images from the CI runs. Will do some more local tests but it seems unlikely it is forking overhead.